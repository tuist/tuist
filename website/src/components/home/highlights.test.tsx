import React from 'react'

import Highlights from './highlights'
import renderer from 'react-test-renderer'

describe('Higlights', () => {
  it('renders the right HTML', () => {
    const tree = renderer.create(<Highlights />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
