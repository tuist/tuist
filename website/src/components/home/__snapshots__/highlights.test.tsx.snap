// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Higlights renders the right HTML 1`] = `
<div
  className="max-w-70 mx-auto flex flex-row"
>
  <article>
    <header>
      Swift manifest
    </header>
    <div>
      Define projects using a simple Swift DSL inside Xcode
    </div>
  </article>
  <article>
    <header>
      Project description helpers
    </header>
    <div>
      Create your own abstrations to make your projects consistent
    </div>
  </article>
  <article>
    <header>
      Scaffold
    </header>
    <div>
      Automate feature creation by generating a target pre-configured with everything you need
    </div>
  </article>
</div>
`;
